kotlin/internal/jvm/compile.bzl

```diff
@@ -162,9 +190,11 @@ def _compiler_directories(ctx):
     """Creates a dict of the necessary compiler directories for generating compile actions"""
     return dict(
         classdir = _output_dir_path(ctx, "jvm", "classes"),
+        abi_classdir = _output_dir_path(ctx, "jvm", "abi_classes"),
```

Comment:
I'll probably want to create a separate dictionary for KotlinKAPT + KotlinCompile actions which will avoid creating directories which are not needed in either case, e.g: no need for `kapt_gen_stub_dir` when running regular `kotlinc`

Reply:
These directories are a bit of cruft due to poor documentation of tree artifacts. They can be removed, as they don't participate in the compilation.

Reply:
Good point, they're internal implementation details of the builder.

Removed.